src/VisualStudio/Xaml/Impl/CodeFixes/AddMissingNamespace/XamlAddMissingNamespacePrefixCodeFixProvider.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+using System.Composition;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeActions;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.Editor.Xaml.Diagnostics;
+using Microsoft.CodeAnalysis.Editor.Xaml.Features;
+
+namespace Microsoft.CodeAnalysis.Editor.Xaml.CodeFixes.AddMissingNamespace
+{
+    [ExportCodeFixProvider(StringConstants.XamlLanguageName, Name = PredefinedCodeFixProviderNames.AddUsingOrImport), Shared]
+    [ExtensionOrder(After = PredefinedCodeFixProviderNames.AddMissingReference)]
+    internal class XamlAddMissingNamespacePrefixCodeFixProvider : CodeFixProvider
```

Comment:
What did the CodeFix stuff need here that wasn't already public? I'd appreciate a comment and a bug tracking it so we can start to chase some of these. Some of the other stuff that is "entirely new" is OK to me, but this particular one seems odd since code fixes are already a public extensibility point.